web/src/style.css

```
@@ -420,7 +420,7 @@ html[data-theme='coffee'] {
 }
 
 @theme {
-  --font-logo: 'Montserrat', sans-serif;
+  --font-logo: 'Montserrat', system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, sans-serif;
```

Stylelint (17.12.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)